packages/core/src/render3/interfaces/i18n.ts

```diff
@@ -408,3 +413,65 @@ export interface IcuExpression {
   cases: string[];
   values: (string|IcuExpression)[][];
 }
+
+// A parsed I18n AST Node
+export type I18nNode = I18nTextNode|I18nElementNode|I18nICUNode|I18nPlaceholderNode;
+
+export type I18nTextNode = {
```

Comment:
I think we can use `interface`s here (which is more common in the code):
```suggestion
export interface I18nTextNode {
```